CMakeModules/FindOpenCL.cmake

```diff
@@ -29,8 +29,38 @@
 # implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 # See the License for more information.
 #=============================================================================
```

Comment:
All this text, up to, but not including the copyright notice, should be removed.